tools/msp_split/splitMSP.xml

```diff
@@ -0,0 +1,39 @@
+<tool id="splitMSP" name="split spectra" version="0.1.0" python_template_version="3.5">
+    <description>Split MSP spectra file</description>
+    <requirements>
+         <requirement type="package" version="0.14.0">matchms</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        python3  $__tool_directory__/splitMSP.py 
+        --filename '$msp_input'
+        --outdir   'output' 
+    ]]></command>
+    <inputs>
+        <param type="data" name="msp_input" format="msp" />
```

Comment:
This parameter should have a help text mentioning for example which keyword is used in order to split the individual spectra from the library file -> In our case, it is the `name` field which is used to identify the individual spectra.

Reply:
A help description has been added.